Source/WebCore/workers/service/context/ServiceWorkerDebuggable.cpp

```diff
@@ -51,17 +51,10 @@ ServiceWorkerDebuggable::ServiceWorkerDebuggable(ServiceWorkerThreadProxy& servi
 {
 }
 
-void ServiceWorkerDebuggable::connect(FrontendChannel& channel, bool isAutomaticInspection, bool immediatelyPause)
+void ServiceWorkerDebuggable::connect(FrontendChannel& channel, bool, bool)
```

Comment:
Can these parameters be removed? I was also wondering if we can completely remove ServiceWorkerDebuggable? That can be done in a follow-up patch, if it is reasonable to remove it.

Reply:
It turns out that I'd need to change some other less relevant call sites if those were removed. Let's do that later then, possibly by removing `ServiceWorkerDebuggable` altogether when the time comes.